GoogleSignIn/Tests/Unit/GIDFakeFetcher.m

```diff
@@ -39,6 +57,15 @@ - (void)beginFetchWithCompletionHandler:(FetchCompletionHandler)handler {
     [NSException raise:NSInvalidArgumentException format:@"Attempted start fetch again"];
   }
   _handler = [handler copy];
+  [self authorizeRequestWithCompletion:handler];
+}
+
+- (void)authorizeRequestWithCompletion:(FetchCompletionHandler)completion {
+  NSMutableURLRequest *mutableRequest = [NSMutableURLRequest requestWithURL:self.request.URL];
+  [self.authorizer authorizeRequest:mutableRequest completionHandler:^(NSError * _Nullable error) {
+    self.fetcherError = error;
```

Comment:
Is `fetcherError` used after being assigned here?

Reply:
Gah no it isn't and it is not needed.